packages/@sanity/types/src/validation/types.ts

```diff
@@ -232,10 +232,6 @@ export type RuleSpecConstraint<T extends RuleSpec['flag']> = ConditionalIndexAcc
  * @public
  */
 export type ValidationContext = {
-  /**
-   * @deprecated Will be removed in the next version! Use `getClient({apiVersion: '2021-06-07'})` instead
-   */
-  client: SanityClient
```

Comment:
Assuming _next version_ means `v3` here? Otherwise this would technically be a breaking change, right?

Reply:
We got some downstream TS compile errors in the validation tests after this was removed. Pushed a fix for those in 4e9badf084